src/TraceEvent/TraceLog.cs

```diff
@@ -25,6 +25,7 @@
 using System.Diagnostics;
 using System.Diagnostics.Tracing;
 using System.IO;
+using System.Linq;
```

Comment:
With the `HashSet` this is not needed, I believe.

Reply:
Good catch, thanks!